README.md

```diff
@@ -35,3 +35,24 @@ Like so:
 - [`font-weight`](./lib/rules/font-weight/README.md): Specify a scale for font-weight.
 - [`line-height`](./lib/rules/line-height/README.md): Specify a scale for line-height.
 - [`space`](./lib/rules/space/README.md): Specify a scale for space.
+
+Ref: [Styled System Keys Reference](https://styled-system.com/theme-specification#key-reference)
+
+# Why?
+
+This plugin can help you create:
+
+- a consistent look and feel for your apps
+- efficient collaboration you and the designers
+
+When designers review apps in the browser they generally use relative terms. For example, "The space between that heading and that paragraph feels too tight, can we make it bigger?"
+
+You can then pick the next value on the scale and be confident that it'll be consistent with the overall design.
```

Comment:
probably also worth noting that the the development o the scale ought to include both disciplines, perhaps with a hat-tip to a constraints-based design authority?

Reply:
Agreed. I'll put something in the list of rules section where we already talk about specifying scales.